yaqd_wright/_wright_filter_wheel.py

```diff
@@ -47,7 +57,7 @@ def _set_microstep(self, microint):
     async def update_state(self):
         while True:
             self._serial_port.write(f"Q {self._motornum}\n".encode())
-            line = await self._serial_port.areadline()
+            line = await self._serial_port.awritethenreadline()
```

Comment:
actual method name is `awrite_then_readline()` with underscores, add the contents of the previous line's write call and delete that write, write_then_readline, as the name suggests, performs the write operation